Dockerfile

```diff
@@ -1,12 +1,22 @@
-FROM registry.opensource.zalan.do/stups/openjdk:latest
+FROM registry.opensource.zalan.do/library/openjdk-8:latest
 
 MAINTAINER Zalando SE
 
+# add AWS RDS CA bundle
+RUN mkdir /tmp/rds-ca && \
+    curl https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem > /tmp/rds-ca/aws-rds-ca-bundle.pem
+# split the bundle into individual certs (prefixed with xx)
+# see http://blog.swwomm.com/2015/02/importing-new-rds-ca-certificate-into.html
+RUN cd /tmp/rds-ca && csplit -sz aws-rds-ca-bundle.pem '/-BEGIN CERTIFICATE-/' '{*}'
+RUN for CERT in /tmp/rds-ca/xx*; do mv $CERT /usr/local/share/ca-certificates/aws-rds-ca-$(basename $CERT).crt; done
+RUN update-ca-certificates
+
 COPY resources/api/kio-api.yaml /zalando-apis/
+COPY target/kio.jar /
+COPY java-dynamic-memory-opts /bin/java-dynamic-memory-opts
```

Comment:
```suggestion
COPY java-dynamic-memory-opts /usr/local/bin/java-dynamic-memory-opts
```

Reply:
Besides that LGTM (sorry I don't have write permissions to commit this suggestion)